### PR TITLE
File names in the commit modal are invisible in dark mode

### PR DESCRIPTION
**Pull request by sanchitmalhotra126**
Added default color to file name label.

Screenshots:
- Dark mode
Screen Shot 2021-07-02 at 11 31 01 AM
- Light mode:
Screen Shot 2021-07-02 at 11 31 12 AM
## Links

https://codedotorg.atlassian.net/browse/CSA-528

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/javalab/CommitDialog.jsx [hunk 1]
```diff
@@ -172,7 +172,8 @@ const styles = {
     borderBottom: `1px solid ${color.lightest_gray}`
   },
   fileLabel: {
-    flexGrow: 2
+    flexGrow: 2,
+    color: color.default_text
```
Comment (hannahbergam) [inline apps/src/javalab/CommitDialog.jsx:176]: Do you have a screenshot you could add?

Comment (sanchitmalhotra126) [reply apps/src/javalab/CommitDialog.jsx:176]: Added

Comment (hannahbergam) [reply apps/src/javalab/CommitDialog.jsx:176]: And what did it look like before?

**Review by hannahbergam — APPROVED** [review]
🙌

File: apps/src/javalab/CommitDialog.jsx (hunk 1 above)
Comment (sanchitmalhotra126) [reply apps/src/javalab/CommitDialog.jsx:176]: Ah yep that's in the JIRA: https://codedotorg.atlassian.net/browse/CSA-528

**Pull request merged**